features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -147,6 +178,35 @@ class WisunBorderRouter {
      * */
     mesh_error_t validate_pan_configuration(uint16_t pan_id);
 
+    /**
+     * \brief Get Wi-SUN Border Router information.
+     *
+     * Function reads RPL information from Border Router.
+     * Mesh interface must be initialized before calling this function.
+     *
+     * \param info_ptr Structure given to stack where information will be stored
+     *
+     * \return MESH_ERROR_NONE on success.
+     * \return MESH_ERROR_UNKNOWN in case of failure.
+     * */
+    mesh_error_t get_info(br_information_t *info_ptr);
```

Comment:
usually naming should follow the convention of data_(method) so should be info_get so in future info_set can be made